openedx/frontend-lib-content-components
# fix: prevent misplaced scrollable shadow when video gallery is loading

## Description

Prevent ugly shadow when video gallery is loading.

Before: ![image](https://github.com/openedx/frontend-lib-content-components/assets/1616648/e88ec682-e922-4849-b2bf-8fb13697f616)

After: ![image](https://github.com/openedx/frontend-lib-content-components/assets/1616648/e0a234a7-2345-43f5-b0b3-a7407442cbb4)

## Testing instructions

1. (Optional) Create a mock videos response with a useful delay
   1. Enable mock video uploads (create [waffle flag](http://localhost:18010/admin/waffle/flag/) `contentstore.mock_video_uploads` enabled for everyone)
   2. Modify [videos_index_json()](https://github.com/openedx/edx-platform/blob/565b34e4e0904d1a55e56db403893a207c62e4c3/cms/djangoapps/contentstore/video_storage_handlers.py#L692) to have a delay so the loading phase can be observed more carefully, as follows:
      ```
      def videos_index_json(course):
          """
          ...
          """
          import time
          time.sleep(20)
          # index_videos, __ = _get_index_videos(course)
          index_videos = [{
              'edx_video_id': 'aaaaaaaa-aaaa-4aaa-aaaa-aaaaaaaaaaaa',
              'client_video_id': 'video.mp4',
              'created': '1970-01-01T00:00:00Z',
              'duration': 42.5,
              'status': 'upload',
              'course_video_image_url': 'https://video/images/1234.jpg'
          }]
          return JsonResponse({"videos": index_videos}, status=200)
      ```
2. Open video gallery
   1. Enable the new video editor (create [waffle flags](http://localhost:18010/admin/waffle/flag/) `new_core_editors.use_new_video_editor` and `new_core_editors.use_video_gallery_flow` enabled for everyone)
   2. Create a course unit with a video
   3. Edit the video (should open in new editor)
   4. Click "< Replace video" to go to the gallery
3. See that the shadow does not appear while the gallery is loading
